AdobeStockImageAdminUi/view/adminhtml/web/js/media-gallery.js

```diff
@@ -56,19 +76,11 @@ define([
             }
 
             //select folder
-            imageFolder = $('.jstree a:contains("' + imageFolderName + '")');
-
-            if (imageFolder.length) {
-                imageFolder[0].click();
-                //select image
-                locatedImage = $('div[data-row="file"]:has(img[alt=\"' + imageFilename + '\"])');
-
-                return locatedImage.length ?  locatedImage : false;
-            }
-
-            $.ajaxSetup({
-                async: true
+            imageFolder = $('.jstree a').filter(function () {
+                return $.trim($(this).text()) === imageFolderName;
             });
+
+            return imageFolder;
```

Comment:
I'd move the following code snippet to this method and renamed it to `selectFolder`

            if (imageFolder.length) {
                imageFolder[0].click();

Reply:
done, please check. Tnx!